lib/ember_cli/shell.rb

```diff
@@ -50,6 +52,23 @@ def test
     attr_accessor :pid
     attr_reader :ember, :env, :options, :paths
 
+    def invalid_ember_dependencies?
+      !run("#{paths.ember} version")
+    rescue DependencyError
```

Comment:
If a `DependencyError` was raised, what makes it more favorable to catch it and return `false` than to let it bubble and crash the program?